test/integration/js/postgres/copy.csv

```diff
@@ -0,0 +1,2 @@
+1	2	3
+4	5	6
```

Comment:
what does it do?

Reply:
No idea, it is copied as is from `postgres.js` test folder. I haven't really checked if that is used anywhere in test suite.

Reply:
let's try to run it without this file :D

Reply:
I have found that it is used in one test:
```js
t('Copy from file', async() => {
  await sql`create table test (x int, y int, z int)`
  await new Promise(async r => fs
    .createReadStream(rel('copy.csv'))
    .pipe(await sql`copy test from stdin`.writable())
    .on('finish', r)
  )

  return [
    JSON.stringify(await sql`select * from test`),
    '[{"x":1,"y":2,"z":3},{"x":4,"y":5,"z":6}]',
    await sql`drop table test`
  ]
})
```